src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  const costAmount = days * PRICE_PER_DAY;
```

Comment:
```suggestion

  const costAmount = days * PRICE_PER_DAY;
```
Consider add some whitespace.

Reply:
> Consider add some whitespace.

done